src/main/kotlin/com/jetbrains/snakecharm/stringLanguage/lang/SmkSLPythonRegExpInjector.kt

```diff
@@ -2,14 +2,23 @@ package com.jetbrains.snakecharm.stringLanguage.lang
 
 import com.intellij.lang.injection.MultiHostRegistrar
 import com.intellij.psi.PsiElement
+import com.intellij.psi.util.PsiTreeUtil
 import com.intellij.psi.util.elementType
 import com.jetbrains.python.codeInsight.PyInjectionUtil
 import com.jetbrains.python.codeInsight.PyInjectorBase
 import com.jetbrains.python.codeInsight.regexp.PythonRegexpLanguage
+import com.jetbrains.python.psi.PyArgumentList
+import com.jetbrains.python.psi.PyCallExpression
+import com.jetbrains.python.psi.PyReferenceExpression
 import com.jetbrains.python.psi.PyStringLiteralExpression
+import com.jetbrains.python.psi.impl.PyKeywordArgumentImpl
+import com.jetbrains.snakecharm.lang.SnakemakeLanguageDialect
+import com.jetbrains.snakecharm.lang.SnakemakeNames.WORKFLOW_WILDCARD_CONSTRAINTS_KEYWORD
+import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpointArgsSection
+import org.jetbrains.annotations.Nullable
 
 open class SmkSLPythonRegExpInjector : PyInjectorBase() {
```

Comment:
Current impl doesn't support all cases mentioned in the documentation, see https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html?highlight=wildcard_constraints#wildcards section

Reply:
I added implementation for global wildcard constraints too